go.mod

```
@@ -1,24 +1,25 @@
 module github.com/cloudogu/k8s-ces-setup
 
-go 1.21
+go 1.22.0
 
-toolchain go1.21.1
+toolchain go1.22.4
 
 require (
-	github.com/cloudogu/cesapp-lib v0.12.1
+	github.com/cloudogu/cesapp-lib v0.12.2
 	github.com/cloudogu/k8s-apply-lib v0.4.2
-	github.com/cloudogu/k8s-component-operator v0.7.0
-	github.com/cloudogu/k8s-dogu-operator v0.38.0
+	github.com/cloudogu/k8s-component-operator v1.0.1
+	github.com/cloudogu/k8s-dogu-operator v1.2.0
+	github.com/cloudogu/k8s-registry-lib v0.2.0
```

```suggestion
	github.com/cloudogu/k8s-registry-lib v0.2.1
```
There is a new Version. Should be a simple update